dracoon/reports/__init__.py

```diff
@@ -71,64 +77,99 @@ async def create_report(self, report: CreateReport, raise_on_err: bool = False)
         except httpx.HTTPStatusError as e:
             self.logger.error("Creating report failed.")
             await self.dracoon.handle_http_error(err=e, raise_on_err=raise_on_err)
-        
+
         self.logger.info("Created report.")
         return None
-    
-    def make_report(self, name: str, target_id: int, formats: List[ReportFormat], type: ReportType = ReportType.single, 
-                    sub_type: ReportSubType = ReportSubType.audit_report, 
-                    enabled: bool = None, filter: ReportFilter = None) -> CreateReport:
-        """ make a report payload to generate a report """
+
+    def make_report(
+        self,
+        name: str,
+        target_id: int,
+        formats: List[ReportFormat],
+        type: ReportType = ReportType.single,
+        sub_type: ReportSubType = ReportSubType.audit_report,
+        enabled: bool = None,
+        filter: ReportFilter = None,
+    ) -> CreateReport:
+        """make a report payload to generate a report"""
         report = {
             "name": name,
-            "target": {
-                "id": target_id
-            },
+            "target": {"id": target_id},
             "type": type,
             "subType": sub_type,
-            "formats": formats  
+            "formats": formats,
         }
 
-        if enabled is not None: report["enabled"] = enabled
-        if filter: report["filter"] = filter
+        if enabled is not None:
+            report["enabled"] = enabled
+        if filter:
+            report["filter"] = filter
 
         return CreateReport(**report)
 
-
-    def make_report_filter(self, from_date: datetime = None, to_date: datetime = None, parent_room_id: int = None, 
-                           user_id: int = None, operations: List[int] = None) -> ReportFilter:
-        """ make an optional report filter needed for make_report() """
+    def make_report_filter(
```

Comment:
@semtisem I'm uncertain if it actually makes sense accepting `datetime` and doing some conversion instead of just accepting a string - thoughts?